libs/MeshKernel/include/MeshKernel/BoundingBox.hpp

```diff
@@ -166,6 +166,9 @@ namespace meshkernel
     /// @brief Merge two bounding boxes into a single bounding box that will contain both of the original.
     BoundingBox Merge(const BoundingBox& b1, const BoundingBox& b2);
 
+    /// @brief Create a non overlapping bounding box
+    BoundingBox CreateEmptyBoundingBox();
+
```

Comment:
Perhaps change name of function, it is already inside the BoundingBox class, so perhaps does not need to have BoundingBox in the name